BackBLAST.py

```diff
@@ -198,7 +199,7 @@ def main(args):
     if not subject_proteome_file.endswith(".faa"):
         print("[Warning] " + subject_proteome_file + " may not be a amino acid FASTA file!")
 
-    out_file = subject_proteome_file.rstrip(".faa") + ".csv"
+    # out_file = subject_proteome_file.rstrip(".faa") + ".csv"
```

Comment:
There is no reason to leave commented out code if you are uses a source control system like Git. Having commented out code is considered bad for a variety of reasons. See: https://markhneedham.com/blog/2009/01/17/the-danger-of-commenting-out-code/